File: Pub-Sub-Pulsar/vendas/src/main/java/com/example/pulsar/vendas/model/Saler.java
package com.example.pulsar.vendas.model;

import java.math.BigDecimal;
import java.util.List;

public class Saler {
    private BigDecimal value;
    private String idStore;
    private List<String> products;
    private String adrres;
    private String idCliente;
    private String idSale;

    public BigDecimal getValue() {
        return value;
    }

    public String getIdStore() {
        return idStore;
    }

    public List<String> getProducts() {
        return products;
    }

    public String getAdrres() {
        return adrres;
    }

    public String getIdCliente() {
        return idCliente;
    }

    public String getIdSale() {
        return idSale;
    }
}

File: SQS-SNS/vendas/src/main/kotlin/com/sqs/service/vendas/model/Saler.kt
package com.sqs.service.vendas.model

import java.math.BigDecimal

data class Saler(
    val value: BigDecimal,

    val idStore: String,

    val products: List<String>,

    val adress: String,

    val idClient: String,

    val idSale: String,

    )
